documentation/modules/creating-migration-quick-plan-new-source.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc
+
+:_content-type: PROCEDURE
+[id="creating-migration-quick-plan-new-source_{context}"]
+= Creating a migration quick plan with a new source provider
+
+You can create a new source proider as part of creating a migration _quick plan_.
+
+.Prerequisite
+
+* There is at least one user-defined destination provider.
+
+.Procedure
+
+. In the {ocp} web console, click *Providers for virtualization* and then click *Create Provider*.
+. Create a source provider following the instructions in "Adding Source providers" TBD
++
+The *Provider details* page opens in the *Details* tab. This might take a few minutes for a vSphere source provider.
+. Click the *Virtual Machines* tab.
+. Select the VMs you want to migrate and then click *Migrate*.
++
+The *Create migration plan* page opens. It includes the source provider's name and suggestions for a plan name, a target provider and namespace, a network map, and a storage map.
+. Make any desired changes to the editable items. Click *Add mapping* to edit a suggested network mapping, edit a storage mapping, or to add additional mappings.
+. Click *Create migration plan*.
++
+{project-short} validates the migration plan and the *Plan detail* page opens,
+indicating whether the plan is ready for use or contains an error. In all cases, the details of the plan are listed and you can edit the items you filled in on the previous page. If you make any changes, {project-short} again validates the plan.
+. If the plan is valid,
+.. You can run the plan now by clicking *Start migration*.
+.. You can run the plan later by selecting it on the *Plans for virtualization* page and following the procedure in "Running Migration Plans" TBD.
+
+
```

Comment:
@anarnold97 This is the same as the first one, so I'll just copy-paste your answer from above. Thanks again.